src/random.jl

```diff
@@ -114,3 +114,69 @@ Return BitVector of primary producers.
 function whoisproducer(A)
     vec(.!any(A, dims=2))
 end
+
+"""
+**Create facilitation links**
+
+Takes trophic matrix (A) and number or connectance of facilitation links (resp. L and C).
```

Comment:
I'm sorry I don't understand. You may either input one or the other? I see no parameter named `C`?